scripts/release/utils.js

```diff
@@ -58,7 +58,7 @@ async function runYarn(args, options) {
   args = Array.isArray(args) ? args : [args];
 
   try {
-    return await execa("yarn", ["--silent", "run", ...args], options);
+    return await execa("yarn", [...args, "--silent"], options);
```

Comment:
Can't we remove `--silent`?

You can also release with `--skip-dependencies-install` https://github.com/prettier/prettier/tree/main/scripts/release#flags